src/python/pants/backend/python/target_types.py

```diff
@@ -264,15 +267,39 @@ class PythonTestsDependencies(Dependencies):
     supports_transitive_excludes = True
 
 
-class PythonRuntimeBinaryDependencies(StringSequenceField):
-    """Addresses to binary targets that will be built and included in this target at runtime.
+class PythonRuntimeBuildDependencies(StringSequenceField):
+    """Addresses to targets that can be built with the `./pants build` goal and whose resulting
+    assets should be included in the test run.
```

Comment:
For some reason (perhaps not a good reason) I found it difficult to understand what
> and whose resulting
>    assets should be included in the test run.

was referring to without checking the pytest code. Could we possibly elaborate just a tiny bit with
```suggestion
    assets should be included in the test run. This allows depending on python_binary() targets which you can then execute in your tests, for example.
```

Reply:
Yes. I will merge the below sentence with this paragraph. Thanks!